enumflags/src/lib.rs

```diff
@@ -69,21 +70,57 @@ mod details {
     impl BitFlagNum for u32 {}
     impl BitFlagNum for u64 {}
     impl BitFlagNum for usize {}
-}
```

Comment:
The helper formatter structs really shouldn't be in this module. What it's for is only the `BitFlagNum` trait - it needs to be a part of public interface, but we don't want anyone implementing it on their own types.